dev/container/README.md

```diff
@@ -1,52 +1,35 @@
-# container
-
+# Ruby version testing
 This directory contains the files that allow experimenting with
 any desired Ruby version locally.
 
 This is especially useful for older, deprecated Ruby versions
 which are often difficult to install due to openssl
 compatibility problems.
 
-# Start Container
+## Start Container
 
 Do the following from the project's root directory:
 
 ```sh
 export RUBY_VERSION=[VERSION]
-export ID=$(id -u)
-docker-compose --file dev/container/compose.yml build
-docker-compose --file dev/container/compose.yml up -d
+docker compose --file dev/imap-compose.yml up -d
```

Comment:
`docker-compose` is v1, the new version uses a syntax without hyphen ([docker docs](https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2))

Reply:
docker, in their infinite wisdom did not make v2 backwards compatible or stub it in. I'll revert to v1